Dockerfile

```diff
@@ -1,5 +1,5 @@
 #Copied from https://github.com/dacort/metabase-athena-driver/blob/d7572cd99551ea998a011f8f00a1e39c1eaa59b8/Dockerfile
-ARG METABASE_VERSION=v0.46.6.2
+ARG METABASE_VERSION=v0.50.26
```

Comment:
yeah, thi suggestion might have been a mistake. I figured that if latest master of relferrerira built againt 0.50.26, this would as well

Reply:
Ah yes, fair assumption tbh